### PR TITLE
add support for stdout/stderr

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

File: yagna_dapp_manager/dapp_manager.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 PathType = Union[str, os.PathLike]
 
 
+from dapp_runner import _util
```
Comment (johny-b) [inline yagna_dapp_manager/dapp_manager.py:19]: I think this is not used now?

**Review by johny-b — APPROVED** [review]
LGTM!

But please remove the unused import from `dapp_runner` (or explain why we have it).

File: yagna_dapp_manager/dapp_manager.py (hunk 1 above)
Comment (shadeofblue) [reply yagna_dapp_manager/dapp_manager.py:19]: that's some glitch

Comment (shadeofblue) [reply yagna_dapp_manager/dapp_manager.py:19]: aaah, not a glitch but a remainder of my "hack" ;p, removing immediately

**Pull request merged**